### ISSUE TITLE
LaTeX equation labeling in markdown

### ISSUE THREAD

**Issue by dojt** [fonsp/Pluto.jl#678]
There seems to be a problem with  equation labeling:
````
md"""
```math
\begin{equation}\label{blahbel}
\partial^2 \approx 7
\end{equation}
```
"""
````
runs correctly the first time it is run, but if you press shift-enter a second time, it gives an error in the LaTeX code.  The error goes away when you change the label, so it seems to come from LaTeX complaining about "multiply defined labels" or whatnot.
![image](https://user-images.githubusercontent.com/57910358/99183269-21e6d480-2743-11eb-896f-aeaf9e326fcc.png)

**Comment by DhruvaSambrani** [comment]
Check if similar to #488

**Comment by zmoon** [comment]
In addition to label change, error also goes away when the page is refreshed.

<details>
<summary>Same thing appears to happen if you do actually use the label twice on the page</summary>

![image](https://user-images.githubusercontent.com/15079414/99253717-40021280-27df-11eb-93c3-1a673809b1ff.png)
</details>

Seems related to MathJax processing, not #488 (which is caused by pre-processing the MathJax with `text{}`). MathJax has already seen that label so complains multiply defined. Something related to this?: http://docs.mathjax.org/en/latest/web/typeset.html#updating-previously-typeset-content

**Comment by fonsp** [comment]
Even if this is fixed, would the `\label` feature be useful?

**Comment by zmoon** [comment]
It could be useful in a notebook with a lot of text and equations.

You use `\ref` or `\eqref`,
<details>
<summary>though the numbering doesn't seem to be working</summary>

![image](https://user-images.githubusercontent.com/15079414/99282770-21614300-2802-11eb-8f2c-1e8c9d6b0349.png)
</summary>

**Comment by fonsp** [comment]
But how do you use `\ref`? Can MathJax generate clickable links inside the svg?

**Comment by fonsp** [comment]
It might still be a good idea to call `typesetClear` https://github.com/fonsp/Pluto.jl/issues/678#issuecomment-727957475

**Comment by zmoon** [comment]
> But how do you use `\ref`? Can MathJax generate clickable links inside the svg?

Yes, blue clickable jump links that take you to the equation. I put an example of usage in `<details>` of my previous comment (`In equation $\ref{label}$...`), though the jumping/numbering doesn't currently work with equation numbering not enabled.

**Comment by CarloLucibello** [comment]
it would be great to be able to cross-reference equations in a notebook, I hope this issue can be solved

**Comment by fonsp** [comment]
http://docs.mathjax.org/en/latest/web/typeset.html#tex-reset looks useful

**Comment by CarloLucibello** [comment]
A workaround was provided in this discussion
https://discourse.julialang.org/t/pluto-how-to-number-equations-in-latex-mode/59900/9

**Comment by disberd** [comment]
@CarloLucibello Thanks for linking the post.
Just clarifying that the solution in the discourse post uses KaTeX (and it has been very shallowly tested) so it can't be directly applied to deal with equation numbering in mathjax using the standard latex synthax from @md_str

**Comment by doppioandante** [comment]
AFAICT now Pluto uses MathJax, has the status of this issue changed somehow?

**Comment by disberd** [comment]
Pluto always used MathJax from what I remember (and the issue is/was related to that). The latest comment on Katex was just for the workaround in the link above (which creates a custom function for rendering latex equations that skips markdown/mathjax and use katex directly

**Comment by rlaunemacedo** [comment]
Experimentem isto:
$$
\begin{align}
  \tag{1.1}
  V_{sphere} = \frac{4}{3}\pi r^3
\end{align}
$$
